client/src/components/Form/Elements/FormText.vue

```diff
@@ -82,31 +81,12 @@ export default {
             type: Array,
             default: null,
         },
-
-        // These last three are for handling special input cases only
-
-        optional: {
-            type: Boolean,
-            default: false,
-        },
-        model_class: {
-            type: String,
-            default: "",
-        },
-        data: {
-            type: Object, // ?
-            default: () => {},
-        },
```

Comment:
Are these possibly important props for some tools/forms?

Reply:
So `optional` was not used, and `model_class` and `data` where only used to decide wether to display the `datalist` feature or not. That seems to be unnecessary, why not just show the `datalist` if the content is available and it is a single input field.